### ISSUE TITLE
[enhancement] Progress bars in calorie count

### ISSUE THREAD

**Issue by lawson85** [davidhealey/waistline#725]
A simila
![Screenshot_20230620-092157](https://github.com/davidhealey/waistline/assets/52282566/16785a83-6663-4007-8bd8-d81be1b7282d)
r app called Macros (https://play.google.com/store/apps/details?id=com.josmantek.macros) which I'm trying to replace with Waistline, shows colourful progress bars for how many calories/fat/carbohydrates consumed for the day and have bigger fonts than Waistline.

**Comment by davidhealey** [comment]
If you tap on the calories in the diary footer you'll see a little progress ring.

**Comment by davidhealey** [comment]
Yeah I want to completely rework the footer area, but I'm not actively working on Waistline so it will probably be a while before it gets done.
